{{ cookiecutter.project_slug }}/requirements/pip-tools.in

```diff
@@ -0,0 +1,5 @@
+# Just the dependencies to run pip-tools, mainly for the "upgrade" make target
```

Comment:
Why do we need this file? Can we just install pip-tools in the upgrade make target? @morenol

Reply:
Yes, we can do that, but then we will have problems if a new version of pip-tools is not backward compatible